gemrb/core/Scriptable/Scriptable.cpp

```diff
@@ -2024,7 +2026,7 @@ int Movable::GetPathLength()
 PathNode *Movable::GetNextStep(int x)
 {
 	if (!step) {
-		DoStep((unsigned int) ~0);
+		DoStep((unsigned int) ~0);	// Called with max speed
```

Comment:
Is the comment still accurate? ~0 was used (as max time, really, the old misnomer) to make sure no step was taken, just one returned.
Or better yet, is this block still needed? Try adding an error() (assert) while we're still working on it, to see if it's commonly hit.